### PR TITLE
(FM-7989) Remove beaker testing.

### PR DESCRIPTION
**Pull request by ThoughtCrhyme**

### CODE REVIEW

**Comment by ThoughtCrhyme** [comment]
Removed beaker tests, passing pipelines in parallel here https://pipelines.puppet.com/team-modules/builds/566774

File: distelli-manifest.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    - bundle exec rake litmus:acceptance:parallel
+  AfterBuildSuccess:
+    - source /opt/rh/rh-ruby25/enable
+    - bundle exec rake litmus:tear_down
+  AfterBuildFailure:
+    - source /opt/rh/rh-ruby25/enable
+    - bundle exec rake litmus:tear_down
+  CommitData:
+    - RepoType: Git
+    - RepoPath: .
```
Comment (michaeltlombardi) [inline distelli-manifest.yml]: Nit: no trailing newline at end of file

File: inventory.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+groups:
+- name: ssh_nodes
+  nodes: []
+- name: winrm_nodes
+  nodes:
+  - name: jie1juz3iq95imb.delivery.puppetlabs.net
```
Comment (michaeltlombardi) [inline inventory.yaml]: Should this be committed at all or should inventory.yaml be in gitignore?

It looks like it's committing a secret.

Comment (ThoughtCrhyme) [reply inventory.yaml]: oops, forgot to add this to my .gitignore

File: appveyor.yml [hunk 3]
```diff
@@ -33,6 +33,27 @@ environment:
...
+  matrix:
+    only:
+      - ACCEPTANCE: yes
+  install:
+    - set PATH=C:\Ruby%RUBY_VERSION%\bin;%PATH%
+    - bundle install --jobs 4 --retry 2
+    - type Gemfile.lock
+  test_script:
+     - bundle exec rake spec_prep
+     - bundle exec rake litmus:parallel:localhost
```
Comment (michaeltlombardi) [inline appveyor.yml]: I was a liar, this should _actually_ be:

```suggestion
     - bundle exec rake litmus:acceptance:localhost
```

Comment (ThoughtCrhyme) [reply appveyor.yml]: https://media1.tenor.com/images/1dc3dd6d61d297f751e95edd897565e5/tenor.gif?itemid=10893296

**Pull request merged**